### ISSUE TITLE
Add attribute to point cloud bug

### ISSUE THREAD

**Issue by ACornuIGN** [Pix4D/pyopf#7]
Hello,

I tested your add attribute code on a point cloud provided on your git (https://github.com/Pix4D/pyopf) on the olympic_flame example you provide. And I noticed a problem with the point cloud visualization. I'm using Blender to visualize the point cloud. Before testing the code, the point cloud was displayed correctly. After adding a new attribute to the point cloud, it no longer appears. 

I noticed that this bug occurred when the output path was in the same document as the base .gltf file. It doesn't occur when the new .gltf is saved in a different folder, but is no longer linked to the .opf project.

Note: another test was carried out with the example in the git of the .opf format specifications (https://github.com/Pix4D/opf-spec/tree/main/examples), and I noticed that with this example certain files on rewriting had a different name, classes.bin becomes class.bin for example.

Have a nice day
Antoine

**Comment by ACornuIGN** [comment]
Here's the code I used with your olympic_flame example

```
import numpy as np
from pathlib import Path
from pyopf.pointcloud import GlTFPointCloud

pcl = GlTFPointCloud.open(Path('/home/.../olympic_flame/dense/pcl.gltf'))

# Generate a new point attribute as a random vector of 0s and 1s
# The attribute must have one scalar per point
new_attribute = np.random.randint(5, 10, size=len(pcl.nodes[0]))
#print(new_attribute,len(pcl.nodes[0]))

# The attribute must have the shape (number_of_points, 1)
new_attribute = new_attribute.reshape((-1, 1))
# Supported types for custom attributes are np.float32, np.uint32, np.uint16, np.uint8
new_attribute = new_attribute.astype(np.uint32)

# Set the new attribute as a custom attribute for the node
# By default, nodes might be missing custom attributes, so the dictionary might have to be created
if pcl.nodes[0].custom_attributes is not None:
    pcl.nodes[0].custom_attributes['point_class'] = new_attribute
else:
    pcl.nodes[0].custom_attributes = {'point_class': new_attribute}

pcl.write(Path('/home/.../olympic_flame/dense/pcl.gltf'))
```
And so if you view the gltf before running the code, it will be visible and once the processing is done, if you view it again (by reopening it) the dots will have disappeared.

**Comment by tomashynek** [comment]
Thanks @ACornuIGN, we tried it on our side using your script above, but also the script example we provided. On our side we are able to open it both before and after in gltf viewers as well as in Blender. 

I do agree that file name change is a bit messy, on the other hand the file names do not matter as long as they are consistent with the main project file. Would it be possible for you to share with us all the output files you get? Thank you!

**Comment by ACornuIGN** [comment]
I agree with you about the file names.

I've just redone the test on my side to get the output files and I didn't have the problem of the point cloud not being displayed after the script. 
I did notice a difference from last time. I reinstalled pyopf and after testing the script I noticed that the extension of the recording files had changed (from .bin to .glbin). This was not the case last month.
Maybe a bug when installing pyopf that I didn't get here.

In any case, thank you and sorry for the inconvenience.
Have a nice day,
